### ISSUE TITLE
is_file(): open_basedir restriction in effect. File(app/plugin/README.md/base.php) is not within the allowed path(s): 

### ISSUE THREAD

**Issue by KindlyFire** [Alanaktion/phproject#120]
```
is_file(): open_basedir restriction in effect. File(app/plugin/README.md/base.php) is not within the allowed path(s): (/var/www/clients/client0/web30/web:/var/www/clients/client0/web30/private:/var/www/clients/client0/web30/tmp:/var/www/phpp.kindlyfire.com/web:/srv/www/phpp.kindlyfire.com/web:/usr/share/php5:/usr/share/php:/tmp:/usr/share/phpmyadmin:/etc/phpmyadmin:/var/lib/phpmyadmin)
```

Wich means open_basedir is not deactivated, how do i do that ?

**Comment by Alanaktion** [comment]
`open_basedir` is a setting in your PHP configuration that cannot be overridden by Phproject, however I believe I've fixed this issue in commit 3495454.

**Comment by KindlyFire** [comment]
Yep, it works now ! Thanks.
